docs/http-api/swagger.yaml

```
@@ -0,0 +1,2496 @@
+        - application/vnd.eventstore.competingatom+xml
+      parameters:
+        - name: stream
+          in: path
+          description: The stream the persistent subscription is on
+          required: true
+          type: string
+        - name: subscription
+          in: path
+          description: The name of the subscription group
+          required: true
+          type: string
+        - name: embed
+          in: query
+          description: Needed
+          required: false
+          type: string
+          enum:
+            - None
+            - Content
+            - Rich
+            - Body
+            - PrettyBody
+            - TryHarder
+      responses:
+        "200":
+          description: OK
+    post:
+      tags:
+        - Subscriptions
+      summary: Update a persistant subscription
+      description: >-
+        You can edit the settings of an existing subscription while it is
+        running. This will drop the current subscribers and will reset the
+        subscription internally.
+      operationId: Update subscription
+      produces:
+        - application/vnd.eventstore.competingatom+json
+        - application/vnd.eventstore.competingatom+xml
+      parameters:
+        - name: stream
+          in: path
+          description: The stream the persistent subscription is on
+          required: true
+          type: string
+        - name: subscription
+          in: path
+          description: The name of the subscription group
+          required: true
+          type: string
+        - in: body
+          name: subscriptionItem
+          description: Subscription to create
+          required: false
+          schema:
+            $ref: "#/definitions/SubscriptionItem"
+      responses:
+        "200":
+          description: Subscription updated
+    put:
+      tags:
+        - Subscriptions
+      summary: Create a persistent subscription
+      description: >-
+        Before interacting with a subscription group, you need to create one.
+        You will receive an error if you attempt to create a subscription group
+        more than once. This requires [admin
+        permissions](/v5/server/access-control-lists).
+      operationId: Create subscription
+      produces:
+        - application/vnd.eventstore.competingatom+json
+        - application/vnd.eventstore.competingatom+xml
+      parameters:
+        - name: stream
+          in: path
+          description: The stream the persistent subscription is on
+          required: true
+          type: string
+        - name: subscription
+          in: path
+          description: The name of the subscription group
+          required: true
+          type: string
+        - in: body
+          name: subscriptionItem
+          description: Subscription to create
+          required: false
+          schema:
+            $ref: "#/definitions/SubscriptionItem"
+      responses:
+        "200":
```

```suggestion
        "201":
```
Creating a persistent subscription can also return:
- 209: conflict
- 401: unauthorized
- 400: bad request